.changeset/modern-mangos-complain.md

```diff
@@ -0,0 +1,6 @@
+---
+'@lit-labs/gen-wrapper-vue': minor
+'@lit-labs/vue-utils': minor
+---
+
+Adds package for Vue wrapper generation
```

Comment:
I think we were going to not add any changesets until the public release of a package, or at the very least only do patch versions so we can start with 0.1.0 for the initial release.